py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -243,7 +243,7 @@ def _initialize_get_var_lss(self):
         self.get_var_lss = interp1d(self.log_lambda_var_func_grid,
                                     var_lss,
                                     fill_value='extrapolate',
-                                    kind='nearest')
+                                    kind='cubic')
```

Comment:
I would also change this in lines 281, 285, 620 and 624. More for consistency that this having an impact on the measurements

Reply:
"Nearest" sounds like what is aimed here. If you feel like it, you can get rid of these interpolations while you're at it. They should not be interpolations in my opinion, but I don't know how deeply they are embedded into the code.

Reply:
They are basically used in the reports in the delta_attributes files, so that we can check problems in the fits. I don't think they are used elsewhere, so yes, if we don't need it when saving these files, we might as well remove the interpolation.